package.json

```
@@ -64,6 +65,7 @@
     "@types/sinon": "9.0.1",
     "mocha": "5.2.0",
     "nyc": "13.0.1",
-    "sinon": "9.0.2"
+    "sinon": "9.0.2",
+    "timezone-mock": "^1.1.1"
```

Please remove hat from the dependency, just to prohibit that a breaking change in a future-version of timezone breaks the conversion. Thank you!

```suggestion
    "timezone-mock": "1.1.1"
```